react/features/conference/components/web/Labels.js

```diff
@@ -68,9 +74,10 @@ class Labels extends AbstractLabels<Props, State> {
      */
     render() {
         const { _filmstripVisible } = this.props;
-        const { filmstripBecomingVisible } = this.state;
+        const { filmstripBecomingVisible, windowWidth } = this.state;
         const { VIDEO_QUALITY_LABEL_DISABLED } = interfaceConfig;
-        const className = `large-video-labels ${
+        const largeVideo = windowWidth > 620 ? 'large-video-labels' : 'large-video-labels-small';
```

Comment:
is that possible to use media query?

Reply:
at this implementation we got:
- user open the page
- he change the window size
- windowWidth is still the same

It's possible to bind window resize, but I think that media query is better.